api-module-library/microsoft-teams/manager.js

```diff
@@ -31,7 +31,7 @@ class Manager extends ModuleManager {
         let teamsParams = {
             client_id: process.env.TEAMS_CLIENT_ID,
             client_secret: process.env.TEAMS_CLIENT_SECRET,
-            redirect_uri: `${process.env.REDIRECT_URI}/microsoft-teams`,
+            redirect_uri: `${process.env.REDIRECT_URI}/microsoft-teams/callback`,
```

Comment:
actually to make it more reusable i think we should add 
`this.redirect_uri= get(params, 'redirect_uri', `${process.env.REDIRECT_URI}/microsoft-teams`);
to constructor, ie line 18

```suggestion
            redirect_uri: instance.redirect_uri,
```

Reply:
Done!